Doesn't compile

After merging your changes @diogoscf, it fails to compile

logs:

```
$ npm run build

> telegram-lang-enforcer@1.0.0 build /Users/bartekpacia/dev/node/telegram-lang-enforcer
> tsc

src/discord/bot_discord.ts:96:23 - error TS2488: Type 'GuildChannelManager' must have a '[Symbol.iterator]()' method that returns an iterator.

96       for (const c of channels) {
                         ~~~~~~~~


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! telegram-lang-enforcer@1.0.0 build: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the telegram-lang-enforcer@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bartekpacia/.npm/_logs/2020-05-04T00_16_53_054Z-debug.log
```


: c

--- a/src/discord/bot_discord.ts
+++ b/src/discord/bot_discord.ts
@@ -92,10 +92,8 @@ export class EnforcingDiscordBot extends DiscordBot.Client {
       }
       console.log("This server has no default channel. Sending to first available channel")
       let channel
-      const channels = guild.channels
-      for (const c of channels) {
-        const channelType = c[1].type
-        if (channelType === "text") {
+      for (const c of guild.channels.cache) {
+        if (c[1].type === "text") {
           channel = c[0]
           break
         }

